File: niddler-example/src/main/java/com/icapps/sampleapplication/api/ExampleXMLApi.java
package com.icapps.sampleapplication.api;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by maartenvangiel on 14/11/2016.
 */
public interface ExampleXMLApi {

	@GET("/bcantoni/yql/master/lorem.ipsum.xml")
	Call<ResponseBody> getMenu();

}

File: niddler-example-android/src/main/java/com/chimerapps/sampleapplication/api/ExampleXMLApi.kt
package com.chimerapps.sampleapplication.api

import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by maartenvangiel on 14/11/2016.
 */
interface ExampleXMLApi {

    @get:GET("/bcantoni/yql/master/lorem.ipsum.xml")
    val menu: Call<ResponseBody>

}
